cms/static/js/spec/views/settings/main_spec.js

```diff
@@ -182,5 +204,103 @@ define([
             AjaxHelpers.expectJsonRequest(requests, 'POST', urlRoot, modelData);
         });
 
+        it('can add learning information', function () {
+            this.view.$(SELECTORS.add_course_learning_info).click();
+            expect('click').not.toHaveBeenPreventedOn(SELECTORS.add_course_learning_info);
+            expect(this.model.get('learning_info').length).toEqual(2);
+            this.view.$(SELECTORS.add_course_learning_info).click();
+            expect(this.model.get('learning_info').length).toEqual(3);
+        });
+
+        it('can delete learning information', function () {
```

Comment:
If you can't delete the first learning info item (which seems a bit odd to me), you should add a test that delete is not an option in that case.